src/selectors/per-thread/stack-sample.js

```diff
@@ -113,15 +113,27 @@ export function getStackAndSampleSelectorsPerThread(
       ProfileData.getCallNodeInfo
     );
 
+  const _getInvertedCallNodeInfo: Selector<CallNodeInfo> =
+    createSelectorWithTwoCacheSlots(
+      threadSelectors.getFilteredThread,
```

Comment:
Yes, that's the goal. At this stage it's not possible because we want to create inverted call nodes for stacks which are used by samples / markers / allocations, and in order to find out which stacks those are, we need to look at the other parts of the thread. But once we no longer need that optimization, we can do what you suggested.